File: java-order-service/src/main/java/com/example/ticket4u/internal/orderItem/domain/entity/OrderItem.java
package com.example.ticket4u.internal.orderItem.domain.entity;

import com.example.ticket4u.pkg.entity.BaseEntity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;
import lombok.experimental.SuperBuilder;

@SuperBuilder
@Getter
@Setter
@ToString
@AllArgsConstructor
@NoArgsConstructor
public class OrderItem extends BaseEntity {
    private String uuid;
    private String ticketUuid;
    private int quantity;
    private String orderUuid;
}

File: go-order-service/internal/order/domain/entity/order_item.go
package entity

type OrderItem struct {
	UUID       string `gorm:"column:uuid;primaryKey;type:varchar(255)" json:"uuid"`
	TicketUUID string `gorm:"column:ticket_uuid;type:varchar(255)" json:"ticket_uuid"`
	Quantity   int    `gorm:"column:quantity;type:int" json:"quantity"`
	OrderUUID  string `gorm:"column:order_uuid;type:varchar(255)" json:"order_uuid"`
	BaseEntity
}

func (OrderItem) TableName() string {
	return "order_items"
}
